README.md

```diff
@@ -1,2 +1,31 @@
 # My-Portfolio
+
 Portfolio
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**This is a my personal portfolio website.**
+
```

Comment:
- Please remove this line. Your effort should be to make your README look professional. 

[OPTIONAL] Please try to include as much headings in this template as possible : https://github.com/microverseinc/readme-template